packages/mobile/patches/react-native-tiktok+1.1.1.patch

```diff
@@ -1662,6 +1662,19 @@ index 627b86d..0000000
 -"Authorization failed" = "Isiqnisekiso sehlulekile";
 -"User canceled authorization"= "Umsebenzisi ukhansele isiqinisekiso";
 -"OK" = "OK";
+diff --git a/node_modules/react-native-tiktok/ios/Tiktok.swift b/node_modules/react-native-tiktok/ios/Tiktok.swift
+index ce31978..af2093e 100644
+--- a/node_modules/react-native-tiktok/ios/Tiktok.swift
++++ b/node_modules/react-native-tiktok/ios/Tiktok.swift
+@@ -7,7 +7,7 @@ class Tiktok: UIViewController {
+     
+   @objc
+   func auth(_ callback: @escaping RCTResponseSenderBlock) {
+-    let scopes = ["user.info.basic,video.list"] // list your scopes
++    let scopes = ["user.info.basic"] // list your scopes
```

Comment:
OOC what's this for?

Reply:
The scopes list is hardcoded into `react-native-tiktok` and we don't need `video.list`. The other option is to add it to our scopes and get a re-review, but I figured this was easier / more correct